tools/create_operator.py

```diff
@@ -0,0 +1,318 @@
+    srcs = ["{lower_op}.py"],
+    srcs_version = "PY3",
+    deps = [ ":base",
+        "//temporian/core/data:duration",
+        "//temporian/core/operators:{lower_op}",
+        "//temporian/implementation/numpy:implementation_lib",
+        "//temporian/implementation/numpy:utils",
+        "//temporian/implementation/numpy/data:event",
+        "//temporian/implementation/numpy/data:sampling",
+    ],
+)
+
+    """
+        )
+
+    # Operator implementation test
+    with open(
+        os.path.join(
+            "temporian",
+            "implementation",
+            "numpy",
+            "operators",
+            "test",
+            lower_op + "_test.py",
+        ),
+        "w",
+        encoding="utf-8",
+    ) as file:
+        file.write(
+            f"""{licence()}
+
+from absl.testing import absltest
+
+import pandas as pd
+from temporian.core.operators.{lower_op} import {capitalized_op}
+from temporian.implementation.numpy.operators.{lower_op} import (
+    {capitalized_op}NumpyImplementation,
+)
+from temporian.implementation.numpy.data.event import NumpyEvent
+
+
+class {capitalized_op}OperatorTest(absltest.TestCase):
+    def setUp(self):
+        pass
+
+    def test_base(self):
+        event_data = NumpyEvent.from_dataframe(
+            pd.DataFrame(
+                {{
+                    "timestamp": [1, 2,3,4],
+                    "a": [1.0, 2.0, 3.0, 4.0],
+                    "b": [5,6,7,8],
+                    "c": ["A", "A", "B", "B"],
+                }}
+            ),
+            index_names=["c"],
+        )
+        event = event_data.schema()
+
+        expected_output = NumpyEvent.from_dataframe(
+            pd.DataFrame(
+                {{
+                    "timestamp": [1, 2,3,4],
+                    "a": [1.0, 2.0, 3.0, 4.0],
+                    "b": [5,6,7,8],
+                    "c": ["A", "A", "B", "B"],
+                }}
+            ),
+            index_names=["c"],
+        )
+
+        # Run op
+        op = {capitalized_op}(event=event, param=1.0)
+        instance = {capitalized_op}NumpyImplementation(op)
+        output = instance.call(event=event_data)["event"]
+
+        self.assertEqual(output, expected_output)
+
+
+if __name__ == "__main__":
+    absltest.main()
+
+"""
+        )
+
+    # Operator implementation test  build
+    with open(
+        os.path.join(
+            "temporian", "implementation", "numpy", "operators", "test", "BUILD"
+        ),
+        "a",
+        encoding="utf-8",
+    ) as file:
+        file.write(
+            f"""
+py_test(
+    name = "{lower_op}_test",
+    srcs = ["{lower_op}_test.py"],
+    srcs_version = "PY3",
+    deps = [
+        "//temporian/core/data:dtype",
+        "//temporian/core/data:event",
+        "//temporian/core/data:feature",
+        "//temporian/core/operators:{lower_op}",
+        "//temporian/implementation/numpy/operators:{lower_op}",
+        "//temporian/implementation/numpy:evaluator",
```

Comment:
I don't think we need the evaluator dependency

Reply:
You are right.
*Removed.